src/utils/string/index.ts

```diff
@@ -1,4 +1,53 @@
 import { format } from "date-fns";
+import moment from "moment";
+import "moment-timezone";
+import get from "lodash/get";
+import { useQuery } from "@apollo/react-hooks";
+import { GetUserSettingsQuery } from "gql/generated/types";
+import { GET_USER_SETTINGS } from "gql/queries";
+
+export const useUserTimeZone = (time) => {
+  const { data: userSettingsData } = useQuery<GetUserSettingsQuery>(
+    GET_USER_SETTINGS
+  );
+  const timezone = get(userSettingsData, "userSettings.timezone", true);
+
+  if (timezone) {
+    return moment(time)
```

Comment:
I think this may be unnecessary if `time` is UTC. the Date library actually knows the timezone offset and will take that into account when you construct a new Date object. 
https://www.w3schools.com/jsref/jsref_gettimezoneoffset.asp

Reply:
It seems like that would set it based on local time, as opposed to setting it based on the timezone that the user set. We want to display it based on the time the user set in the preferences. (or maybe not?)

Reply:
I'm thinking that if a user is in SF, but working with a NY based team, they may want to see it displayed in Eastern Time. 
However, if this is not done anywhere else in the codebase, then maybe I should do that for consistency. Thoughts?

Reply:
nvm, doing more research, I think I'm wrong about that